geometry_msgs/Point point
---
bool collision
